### ISSUE TITLE
image url - second stream, more configurable?

### ISSUE THREAD

**Issue by ozett** [henry-spanka/node-red-contrib-hikvision#9]
as i scrolled through your code, looking for a polling timeout-parameter,
i found a hardcoded image-url .
https://github.com/henry-spanka/node-red-contrib-hikvision/blob/master/hikvision.js#L143

i dont use the image-node, but i use a differnent url to access my second-stream in latest firmware:

http://192.168.1.115/Streaming/channels/102/httppreview

may you want to check on this/and or make it configurable in the node?

**Comment by henry-spanka** [comment]
So you basically want a low resolution snapshot, right? What's the difference between httppreview and Picture (...channels/102/Picture)?

**Comment by ozett** [comment]
difference url may be caused by firmware,

or by basic-auth or digest-auth..
dont know exactly, there are lots of different information about snapshot-urls in the web.
also the onvif-way could be an alterantive..but also not clearer to get the urls for snapshots..

**Comment by henry-spanka** [comment]
Meaning currently the image node does not work for you?

**Comment by ozett** [comment]
did not test it, because it startet with http-node grabbing the image.
worked for me, i stayed with it up to now.

if the hikvision-alarm nodes run stable, i will give it a try.
but my node-red must run stable over month...thats the goal...
the hikvision-node is the first one, wich infected stability after 2 years running node-red...
hope we nail that down...
🎉

**Comment by ozett** [comment]
some explantion of different urls for different firmware

https://www.biesalski.info/wordpress/hikvision-kamera-stream-abrufen/

**Comment by henry-spanka** [comment]
Yes that's for streams. But we're only taking an image with max resolution. I could add an option to allow taking an image at low resolution if you need that.

**Comment by ozett** [comment]
why not, if you dont mind?

**Comment by henry-spanka** [comment]
Has been implemented. You can now change the streaming url by passing the parameter "snapshotPath" to the node.

**Comment by ozett** [comment]
👍

**Comment by ozett** [comment]
something was also done to the underlying lib here:

`https://github.com/trave/node-hikvision-api/commit/a4fc1f68af019c49249cc6247d9eb8f4a526209c`

maybe worth to collect improvements from the forks. i counted about 5 little things, that others improved...
